presto-docs/src/main/sphinx/release/release-0.258.rst

```diff
@@ -0,0 +1,44 @@
+=============
+Release 0.258
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix issues in SQL functions where query might fail with compiler error when function implementation includes lambda variable that has the same name as column names or function input names.
+* Fixes a bug in Parquet dereference pushdown. The issue causes column name not picked up properly, resulting in inconsistent query result in certain cases.
+* Add Cauchy distribution CDF :func:`cauchy_cdf` and inverse CDF :func:`inverse_cauchy_cdf` functions to MathFunctions.
+* Add SQL functions :func:`array_dupes` and :func:`array_as_dupes`.
+* Add additional details to memory exceeded error messages to simplify debugging. Disabled by default. Can be enabled by setting the ``verbose_exceeded_memory_limit_errors_enabled`` session property to ``true``.
+* Allow dynamic filtering for large number of collected values by converting the predicate into a range.
```

Comment:
what is dynamic filtering? what's "a large number of collected values"?  Also, don't need the implementation detail of how this was done.

Reply:
huh. didn't know we had support for dynamic filtering for joins in the first place, so didn't expect to see it's support extended.  when was that added?

Or if it's new from this release, we should only have one release note about it with the full description of what's supported.

Reply:
This looks like another cherry-pick from trinodb which is on top of a new feature introduced about dynamic filtering. I will merge it with the original dynamic filtering note.

Reply:
Looking at further, there are bunch of other configs for dynamic filtering as well. So doesn't make sense to just include this and exclude others. Will keep this as a generic note about dynamic filtering only.